From 0c92a1152843384f45d399528762c9aa20a1aa14 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Sat, 30 Oct 2021 09:44:52 +0000
Subject: [PATCH 30/30] WIP: boards: amlogic: add CONFIG_DM_ETH to Radxa Zero

Radxa Zero does not have on-board Ethernet hardware but the common
Amlogic board/SoC files have dependencies on Ethernet code, so we
see the driver-model migration warning during u-boot compile. This
adds the required CONFIG_DM_ETH to the board config, which does no
harm and silences the warning.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 configs/radxa-zero_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/radxa-zero_defconfig b/configs/radxa-zero_defconfig
index acd13f1c6b..88a984bb53 100644
--- a/configs/radxa-zero_defconfig
+++ b/configs/radxa-zero_defconfig
@@ -29,7 +29,7 @@ CONFIG_MMC_MESON_GX=y
 CONFIG_MTD=y
 CONFIG_DM_MTD=y
 # CONFIG_PHY_REALTEK is not set
-# CONFIG_DM_ETH is not set
+CONFIG_DM_ETH=y
 CONFIG_DM_MDIO=y
 CONFIG_DM_MDIO_MUX=y
 # CONFIG_ETH_DESIGNWARE_MESON8B is not set
-- 
2.17.1

